### ISSUE TITLE
[Code Style] Can we use .editorconfig to format the codes?

### ISSUE THREAD

**Issue by LINGLUOJUN** [apache/hertzbeat#2594]
### Question

Since  almost all of us use idea as a tool in our daily development, so I think it's importance of us to support a unify . format file  by .editorconfig that can be imported by idea and more easy to format our code.

**Comment by tomsun28** [comment]
> Another one is that can we ignore the order of imported class? In my oppion,it's no need to care about that

how about this, @crossoverJie the default import order of idea is different from what we configured. I met several developers who asked about the related errors and they could only adjust it manually.

**Comment by tomsun28** [comment]
> Can we use .editorconfig to format the codes and ignore the order of imported class

hi, @LINGLUOJUN of course, the fronted `wep-app` has used the .editorconfig, we can add one in backend.

**Comment by crossoverJie** [comment]
> > Another one is that can we ignore the order of imported class? In my oppion,it's no need to care about that
> 
> how about this, @crossoverJie the default import order of idea is different from what we configured. I met several developers who asked about the related errors and they could only adjust it manually.

Yes, some developers have also been asking me about this issue. I think maybe we should temporarily disable the [import order check](https://github.com/apache/hertzbeat/blob/ab52a497237684904feba1e71f6bdf1167977583/script/checkstyle/checkstyle.xml#L299) and enable it later once HertzBeat is stable.

**Comment by tomsun28** [comment]
> Yes, some developers have also been asking me about this issue. I think maybe we should temporarily disable the [import order check](https://github.com/apache/hertzbeat/blob/ab52a497237684904feba1e71f6bdf1167977583/script/checkstyle/checkstyle.xml#L299) and enable it later once HertzBeat is stable.

+1
